README.md

```diff
@@ -47,6 +47,7 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
 - [Daniel Jenkins](#daniel-jenkins) - **streaming:** JavaScript, PHP, C#, Unity, Game Development, Web Development
 - [Daniel Shiffman](#daniel-shiffman) - **streaming:** JavaScript, Processing, P5.js, WebGL, Machine Learning, Algorithms
 - [David Poindexter](#david-poindexter) - **streaming:** Node.js, JavaScript, React, Docker, AWS, Terraform, DevOps, Developer Tooling
+- [David Pedersen](#david-pdrsn) - **streaming:** Rust, Ruby, Vim
```

Comment:
This anchor link (i.e., `#david-pdrsn`) won't work. These anchor links are auto-generated by GitHub. If you want this to link properly to the section for your name, then the anchor needs to match the heading name below. `#david-pedersen` would be the correct anchor link.

Reply:
Sorry about that. It should be fixed in https://github.com/bnb/awesome-developer-streams/commit/aa3350dec400d5b277257acaee40e17b5d668cf4. I force pushed to my fork's branch fyi.

Thanks for the fast response 😊